addons/website/static/src/snippets/s_hr/000.scss

```diff
@@ -3,14 +3,15 @@
     line-height: 0;
     hr {
         padding: 0;
+        // TODO we already change the way the color is applied (using a border)
+        // in website.css. In master, we should just let that system work the
+        // way it is and not force something here.
```

Comment:
@qsm-odoo Do I merge this forward-port or you want to fix this here?

Reply:
I'll adapt :)